app/app.module.ts

```diff
@@ -19,8 +19,10 @@ import { HelpComponent } from './help/help.component';
 import { CoreModule } from './core/core.module';
 import { FooterComponent } from './footer/footer.component';
 import { AdminModule } from './admin/admin.module';
+import { SharedModule } from './shared/shared.module';
 import { EqualValidatorDirective } from './authentication/signup/equal-validator.directive';
 import { ManageUserComponent } from '../app/userprofile/userprofile.component';
```

Comment:
please read my comment. It does not say anything about removing the shared module ....